### PR TITLE
Show sub accounts in sidebar if any

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/polkadot-js/apps/issues/3224

Looks like this:
image

image

### CODE REVIEW

File: packages/page-accounts/src/Sidebar/Identity.tsx [hunk 1]
```diff
@@ -90,6 +92,20 @@ function Identity ({ address, identity }: Props): React.ReactElement<Props> | nu
                 </div>
               </div>
             )}
+            {!!subs?.length && (
+              <div className='tr subs'>
+                <div className={subs.length > 1 ? 'th top' : 'th'}>{t<string>('subs')}</div>
```
Comment (Tbaut) [inline packages/page-accounts/src/Sidebar/Identity.tsx:97]: Can't remember how you go about plurals. If you cater for it or not. I'm happy to add a condition and have "sub" and "subs"

Comment (Tbaut) [reply packages/page-accounts/src/Sidebar/Identity.tsx]: Thinking about it, it's actually much better, committing that.

File: packages/page-accounts/src/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -249,7 +253,7 @@ export default React.memo(styled(FullSidebar)`
     > * {
       margin-bottom: 0.4rem;
 
-      &:not(:first-child) {
+      &:not(:first-sub) {
```
Comment (jacogr) [inline packages/page-accounts/src/Sidebar/Sidebar.tsx:256]: ? Not sure about this...

Comment (Tbaut) [reply packages/page-accounts/src/Sidebar/Sidebar.tsx:256]: nice catch, sorry about that, a search&replace that went wrong.
```suggestion
      &:not(:first-child) {
```

**Comment by jacogr** [comment]
So not 100% sure about the first one in your example, since we don' actually show those as linked. But I think it has grown on me, since it is almost like a identity easter-egg where we do show relationships. So it is actually cool.

Have some issues with this -

image

And there are a lot of those. So would suggest making -

- the subs expandable with `subs (<count>)`
- move them to the bottom (below fields like website)

**Comment by Tbaut** [comment]
Looks like this now:
image

File: packages/page-accounts/src/Sidebar/Identity.tsx [hunk 3]
```diff
@@ -22,6 +23,7 @@ function Identity ({ address, identity }: Props): React.ReactElement<Props> | nu
   const { api } = useApi();
   const { isRegistrar, registrars } = useRegistrars();
   const [isJudgementOpen, toggleIsJudgementOpen] = useToggle();
+  const subs = useCall<[BalanceOf, AccountId[]]>(api.query.identity.subsOf, [address])?.[1];
```
Comment (jacogr) [inline packages/page-accounts/src/Sidebar/Identity.tsx:26]:
```suggestion
  const subs = useCall<[BalanceOf, AccountId[]]>(api.query.identity?.subsOf, [address])?.[1];
```

**Comment by Tbaut** [comment]
> Almost forgot...

I'm pretty sure I'll forget every single time if there's no TS error on that. Do you know why there isn't any?

**Comment by jacogr** [comment]
The base Substrate includes it. The issue it that not all chains do. :(

**Comment by Tbaut** [comment]
Ah sure, I'll make sure to keep that in mind as a fail safe for later.

**Comment by jacogr** [comment]
Always a bitch,  also step into it, but the major source of issues - verifying it works is easy, verifying it doesn't break, is difficult -

https://github.com/polkadot-js/apps/pull/3299/files
https://github.com/polkadot-js/apps/pull/3149/files

**Pull request merged**